app/assets/javascripts/pageflow/slideshow/atmo.js

```diff
@@ -96,7 +86,7 @@
         crossFade: true,
         playFromBeginning: false,
         rewindOnChange: true,
-        pauseInBackground: true
+        pauseInBackground: false
```

Comment:
This is intended to stop audio playback when mobile browser is sent to background. Is this no longer needed?